### DISCUSSION TITLE
Card recommendations

### DISCUSSION THREAD

**Discussion by maedball** [ekutner/home-connect-hass#201] [Community Support]
Hiho,

does someone have any recommendations for lovelace cards to be used with the integration?

**Comment by alexeiw123** [comment]
No but I'm interested in what you find. I'm yet to set up a 'laundry' dashboard but obviously this would be key. Being able to see state, program, time remaining etc.

**Comment by rickyhoots** [comment]
Am switching to this integration soon (getting new equipment soon) and am thinking of doing this: https://github.com/ShunichiSan/HomeAssistantApplianceCard

For the quick view on a dashboard
